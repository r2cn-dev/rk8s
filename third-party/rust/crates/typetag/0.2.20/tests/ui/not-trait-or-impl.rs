#[typetag::serde]
const HUH: () = ();

fn main() {}
