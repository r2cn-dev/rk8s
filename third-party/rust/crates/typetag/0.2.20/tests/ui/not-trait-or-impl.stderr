error: expected trait or impl block
 --> tests/ui/not-trait-or-impl.rs:2:1
  |
2 | const HUH: () = ();
  | ^^^^^
